internal/pkg/manifest/workload_ecs_test.go

```diff
@@ -218,13 +218,28 @@ func TestSecret_UnmarshalYAML(t *testing.T) {
 			in:        "key: value",
 			wantedErr: errors.New(`cannot marshal "secret" field to a string or "secretsmanager" object`),
 		},
-		"should be able to unmarshal an SSM parameter name": {
-			in:     "/github/token",
-			wanted: Secret{from: aws.String("/github/token")},
+		"should be able to unmarshal an plain SSM parameter name": {
+			in: "/github/token",
+			wanted: Secret{from: stringOrFromCFN{
+				Plain: aws.String("/github/token"),
+			},
+			},
 		},
-		"should be able to unmarshal a SecretsManager ARN": {
-			in:     "arn:aws:secretsmanager:us-west-2:111122223333:secret:aes128-1a2b3c",
-			wanted: Secret{from: aws.String("arn:aws:secretsmanager:us-west-2:111122223333:secret:aes128-1a2b3c")},
+		"should be able to unmarshal an imported SSM parameter name from other cloudformation stack": {
+			in: `from_cfn: "mygithubtoken"`,
+			wanted: Secret{from: stringOrFromCFN{
+				FromCFN: fromCFN{
+					Name: aws.String("mygithubtoken"),
+				},
+			},
```

Comment:
nit: to make it more realistic and show how this field is typically used
```suggestion
			in: `from_cfn: "stack-SSMGHTokenName"`,
			wanted: Secret{from: stringOrFromCFN{
				FromCFN: fromCFN{
					Name: aws.String("stack-SSMGHTokenName"),
				},
			},
```